### ISSUE TITLE
Add accessibility statement for Baksia

### ISSUE THREAD

**Issue by sorensensig** [digdir/baksia#679]
**Is your feature request related to a problem? Please describe.**
Baksia does not have an updated accessibility statement (tilgjengelighetserklæring).

**Describe the solution you'd like**
Copy the statement template from other Digdir products and conduct a wcag/accessibility check to update the statement.

Here's the statement: Bokmål: https://uustatus.no/nb/erklaringer/publisert/876433ca-b225-43b3-8593-cae1d7c29a79
Nynorsk; https://uustatus.no/nn/erklaringer/publisert/876433ca-b225-43b3-8593-cae1d7c29a79

**Comment by andreas-rafaelsen** [comment]
@altinnadmin tar et møte med Olav for å sjekke ut denne.

**Comment by altinnadmin** [comment]
@olavflar er på saken! 🥇

**Comment by olavflar** [comment]
Har startet med kartlegging her;

https://digdir-my.sharepoint.com/:x:/r/personal/olav_friisberg_larssen_digdir_no/Documents/Hjelpdokument%20til%20testing%20for%20tilgjengelighetserkl%C3%A6ring%20-%20baksia.xlsx?d=w985880fb9d9d4dfcb1f988cd907fb00d&csf=1&web=1&e=CwTZQf

**Comment by sorensensig** [comment]
Issues created as part of the WCAG check related to this issue.
(🚧 Work in progress 🚧) 
- #917 
- #918 
- #919 
- #921

**Comment by sorensensig** [comment]
@olavflar have published the [accessibility statement for baksia](https://uustatus.no/nb/erklaringer/publisert/876433ca-b225-43b3-8593-cae1d7c29a79). @altinnadmin, I tried to create a new link in the footer myself, but don't understand where to make the change. Can you have a look at this?
